python/tests/catalog/test_rest.py

```diff
@@ -139,7 +139,7 @@ def test_list_tables_404(rest_mock: Mocker):
 
 def test_list_namespaces_200(rest_mock: Mocker):
     rest_mock.get(
-        f"{TEST_URI}v1/namespaces",
+        f"{TEST_URI}v1/namespaces?parent=",
```

Comment:
I think that we should omit parent if there is no parent. Including it makes it unclear whether this is asking for the root or a parent named `""`.

Reply:
I don't think there is such a thing as a `""` namespace in the spec, since you wouldn't be able to fetch the properties that namespace: `/v1/{prefix}/namespaces//properties:`. I've updated the code to have this distinction.